src/actions/vstudio/vs2010_vcxproj.lua

```diff
@@ -892,11 +893,26 @@
 	end
 
 
+	function m.filterEmpty(dirs)
```

Comment:
This would actually make a useful addition to the table.* namespace; I can think of other places I'd use it.